plugins/Morpheus/stylesheets/general/_forms.less

```diff
@@ -72,23 +72,50 @@ input.browser-default {
   background-color: @theme-color-brand !important;
 }
 
-[type="checkbox"]:checked+label:before {
+[type="checkbox"]:checked + label::before {
```

Comment:
need to check this doesn't break anything

Reply:
See other comment.